docs/nx-cloud/enterprise/dte/jenkins-dte.md

```diff
@@ -0,0 +1,79 @@
+# Custom Distributed Task Execution on Jenkins
+
+Using [Nx Agents](/ci/features/distribute-task-execution) is the easiest way to distribute task execution, but it your organization may not be able to use hosted Nx Agents. With an [enterprise license](https://nx.app/enterprise), you can set up distributed task execution on your own CI provider using the recipe below.
+
+## Distribute Tasks Across Custom Agents on Jenkins
+
+Run agents directly on Jenkins with the workflow below:
+
+```groovy
+pipeline {
+    agent none
+    environment {
+        NX_BRANCH = env.BRANCH_NAME.replace('PR-', '')
```

Comment:
Same as above, add `e2e-ci` and the missing `NX_CLOUD_DISTRIBUTED_EXECUTION_AGENT_COUNT: 3` env variable